lib/testing/test_runner.sh

```diff
@@ -69,6 +69,18 @@ OUTSIDE_MAC=
 ENABLE_IPV4=0
 ENABLE_VLAN=0
 
+version_greater_or_equal()
+{
+    printf '%s\n%s\n' "$2" "$1" | sort -V -C
+}
+
+assert_kernel_version_greater_or_equal()
```

Comment:
Can we not change the naming to skip_test_for_kernel_versions_lower_than() or something, as assert sound more than an abort to me...